update readme (#24)
Bitshala-Incubator/silent-pay-wallet: README.md

# Shroud Wallet

⚠️ **Work in Progress** - This wallet is currently under active development.

> A non-custodial, Bitcoin-only mobile wallet focused on Silent Payments (BIP-352).

Reviewer: ```suggestion
A non-custodial, Bitcoin-only mobile wallet with support for Silent Payments (BIP-352).
```

## Related Projects

This wallet is built using the following sister repositories:

- **[silent-pay](https://github.com/Bitshala-Incubator/silent-pay)** - Core Silent Payments library implementing BIP-352 protocol
- **[silent-pay-indexer](https://github.com/Bitshala-Incubator/silent-pay-indexer)** - Indexer service for efficient Silent Payment transaction scanning

## Getting Started

### Prerequisites
- Node.js >= 20
- npm
- Android Studio (for Android development)
- Xcode (for iOS development)

> ### Installation

Reviewer: lets keep the original build instructions. They are correct right?
Atleast iOS instructions are correct.
